### DISCUSSION TITLE
Regular standup calls across communities

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#699] [Ideas] [answered]
It went so well, let's do more!

<img width="1434" alt="Screenshot 2021-01-17 at 16 12 54" src="https://user-images.githubusercontent.com/624760/104849187-595b3400-58e0-11eb-9459-ddf3aaab5fb8.png">

**Comment by mrunankpawar** [comment, marked as answer]
We can plan a call once every month and share what we achieved and challenges we faced throughout the month.

**Reply by carrycooldude** [reply]
Yaa, I love the Idea

**Reply by ankurg132** [reply]
How about doing that weekly? We'll have a fun weekend this way :)

**Reply by sanyogthescholar** [reply]
If it's not possible weekly, then we can do it bi-weekly. We can have a Discord server where all of us can connect, chat and team up on projects and hackathons. A bi-weekly call will be great to meet new people and show off your projects.
Edit: There's already a Discord server. Link is on https://eddiejaoudecommunity.github.io/

**Reply by eddiejaoude** [reply]
I am planning to do this monthly, starting very very soon 🤓

**Reply by eddiejaoude** [reply]
I did say "very very soon", it is here 🎉 , hopefully see you all tomorrow https://www.eddiehub.org/community-call-sign-up?r_done=1

**Reply by sanyogthescholar** [reply]
> I did say "very very soon", it is here 🎉 , hopefully see you all tomorrow https://www.eddiehub.org/community-call-sign-up?r_done=1

That was quick! Thank you

**Comment by carrycooldude** [comment]
Already Excited for this.

**Reply by eddiejaoude** [reply]
YAY 🎉 me too!
